website/docs/d/dns_custom_resolvers.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+subcategory: "DNS Services"
+layout: "ibm"
+page_title: "IBM : Custom Resolvers"
+description: |-
+  Manages IBM Cloud Infrastructure Private DNS Custom Resolvers.
+---
+
+# ibm_dns_custom_resolvers
+
+Provides a read-only data source for Custom Resolvers. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.For more information about Custom Resolver, refer to [List custom resolvers](https://cloud.ibm.com/apidocs/dns-svcs#list-custom-resolvers).
+
+
+## Example usage
+
+```terraform
+data "ibm_dns_custom_resolvers" "example-cr" {
+  instance_id = "resource_instance_guid"
+}
+```
+
+## Argument reference
+The following arguments are supported: 
+
+- `instance_id` - (Required, String) The GUID of the private DNS service instance.
+
+## Attribute reference
+In addition to the argument references list, you can access the following attribute references after your data source is created. 
+
+- `id`- (String) The unique identifier of the custom resolvers.
+- `custom_resolvers` (List) List of all private DNS custom resolvers.
+ 
+   Nested scheme for `custom_resolvers`:
+   - `custom_resolver_id` - (String) Identifier of the  custom resolver.
+   - `description` - (String) Descriptive text of the custom resolver.
```

Comment:
Why custom_resolver_id as identifier?  Why not "id" like most of the ibm_is_ resources or ibm_dns_custom_resolver
see ibm_is_subnet ibm_is_vpc ibm_is_instance ...

Reply:
updated